src/components/pages/CaseTable/CaseTable.js

```diff
@@ -565,6 +565,18 @@ export default function CaseTable(props) {
             Download All Selected
           </Button>
 
+          <Button
+            style={{
+              background: '#215589',
+              color: '#fff',
+              textTransform: 'uppercase',
+              marginLeft: '.5%',
+            }}
+            type="default"
+          >
+            <PDFExportButton caseData={filter(caseData)} viz={<PieChart />} />
+          </Button>
+
```

Comment:
In the future, I would probably just slap an ant design button in here :)

Reply:
I agree. It may be hard to know because of the mix of everything in here, but we're trying to stick with AntD whenever we have to style something new.